staff/daniel-villegas/cv/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The CV</title>
+</head>
+<body>
+    
+    <section>
+                <font face="verdana" color="blue"><h1>Curriculum</h1></font>
```

Comment:
formatting and styling is NOT part of this challenge (remove it)

Reply:
changed! :)